test_conformance/conversions/basic_test_conversions.h

```diff
@@ -341,7 +355,8 @@ struct IterOverSelectedTypes : public TestType
         {
             // run selected conversion
```

Comment:
This needs some extra checks now too.  Something like:

```suggestion
            if (In != inType) return;
            if (Out != outType) return;

            // run selected conversion
```

Reply:
Comment above applies here as well.

Reply:
Good find - that is correct, I will put additional checks in `testType` function template.

Reply:
Done (through additional condition in `testType` function template)